libs/components/src/lib/dial-pad/dial-pad.ts

```diff
@@ -0,0 +1,200 @@
+import { applyMixins, FoundationElement } from '@microsoft/fast-foundation';
+import { attr } from '@microsoft/fast-element';
+import {
+	Localized
+} from '../../shared/patterns';
+import { TextField } from '../text-field/text-field';
+import { Button } from '../button/button';
+
+/**
+ * Base class for dial-pad
+ *
+ * @public
+ * @event change - Emitted when the text field value changes
```

Comment:
We now annotate component like this for the wrappers:
```suggestion
 * @public
 * @component dial-pad
 * @event change - Emitted when the text field value changes
```